doc/src/reference.xml

```diff
@@ -561,13 +561,15 @@ path-constant DATA : data/a.txt ;
       <varlistentry>
         <term><literal>cflags</literal></term>
         <term><literal>cxxflags</literal></term>
+        <term><literal>compileflags</literal></term>
```

Comment:
OK, I understand the distinction now, and where my previous confusion occurred.

Having the exact same feature name meaning different things depending on where it is used seems to me a really bad programming design decision for Boost Build.

I will remove the change I made, realizing it is wrong.